.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "2.7"
+  - "3.5"
+install:
+  - "pip install -r requirements.txt"
+  - "pip install flake8"
```

Comment:
Should we create a test-requirements.txt, as with our other repos, and add flake8 to it?